src/config.c

```diff
@@ -1016,7 +1016,6 @@ void configGetCommand(client *c) {
 /* We use the following dictionary type to store where a configuration
  * option is mentioned in the old configuration file, so it's
  * like "maxmemory" -> list of line numbers (first line is zero). */
```

Comment:
Somehow this comment got moved from it's original target.  I think it belongs with `optionToLineDictType` (below)